modules/shimmer/1.0/etc/merge_vs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I see no harm in keeping this one though, since it's not a vcf and might be handy if the --regions flag is ever fixed in future releases.